<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAxisDialog</class>
 <widget class="QDialog" name="EditAxisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>331</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="LeftLabels">
       <item>
        <widget class="QLabel" name="XAxisNameLabel">
         <property name="text">
          <string>X-axis name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="YAxisNameLabel">
         <property name="text">
          <string>Y-axis name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="XAxisLimitsLabel">
         <property name="text">
          <string>X-axis Limits</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="XLogCheckBox">
         <property name="text">
          <string>X-axis Logarithmic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="YAxisLimitsLabel">
         <property name="text">
          <string>Y-axis Limits</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="YLogCheckBox">
         <property name="text">
          <string>Y-axis Logarithmic</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>11</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item alignment="Qt::AlignmentFlag::AlignRight">
        <widget class="QLineEdit" name="XAixsName"/>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignRight">
        <widget class="QLineEdit" name="YAisxName"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="XAxisLimitsHorLayout">
         <item>
          <widget class="QLineEdit" name="XLeftLimit">
           <property name="maximumSize">
            <size>
             <width>74</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="XtoLabel">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>To </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="XRightLimit">
           <property name="maximumSize">
            <size>
             <width>74</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="XLogSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="YAxisLimitHorLayout">
         <item>
          <widget class="QLineEdit" name="YLeftLimit">
           <property name="maximumSize">
            <size>
             <width>74</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="YtoLabel">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>To </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="YRightLimit">
           <property name="maximumSize">
            <size>
             <width>74</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="YLogSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditAxisDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditAxisDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
